### PR TITLE
Fix: Remove old projects, format and organize markdown

### PR DESCRIPTION
**Pull request by seangwright**
### Motivation

- Format markdown with markdown lint
- Simplify main `README.md` so it's easier to understand
- Remove old, unused `Migration.Tool.Model` project
- Update GitHub issue template with instructions to collect log information

### Checklist

- [X] Code follows coding conventions held in this repo
- [ ] Automated tests have been added
- [X] Tests are passing
- [X] Docs have been updated (if applicable)
- [ ] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### How to test

If manual testing is required, what are the steps?

### CODE REVIEW

**Comment by seangwright** [comment]
@tkrch I noticed the `Migration.Tool.Model` project was not referenced in the docs, the `.sln` file, or by any other code so I removed it from the repository as part of this docs clean up.

Do you know if this project is still needed? I can revert the deletion if we need to keep it.

File: Migration.Tool.CLI/MIGRATION_PROTOCOL_REFERENCE.md [hunk 1]
```diff
@@ -1,16 +1,15 @@
-## Migration Protocol Reference
+# Migration Protocol Reference
 
 Running the `Migration.Tool.CLI.exe migrate` command (
 see [`Migration.Tool.CLI/README.md`](/Migration.Tool.CLI/README.md)) generates a **migration protocol file**. The
 protocol provides information about the result of the migration, lists required manual steps, etc.
 
-You can find the protocol file in the location specified by
-the `Settings.MigrationProtocolPath` [configuration option](/Migration.Tool.CLI/README.md#Configuration).
+You can find the protocol file path in the `.\Migration.Tool.CLI\appsettings.json` file under the [`MigrationProtocolPath` configuration option](/Migration.Tool.CLI/README.md#Configuration).
```
Comment (Honzaless) [inline Migration.Tool.CLI/MIGRATION_PROTOCOL_REFERENCE.md:7]: We usually try to write the documentation in a task-oriented style. Although finding the protocol file path is a task we'd like the user to achieve, it is not the primary goal, the goal we'd like them to achieve is to find the protocol file.

File: Migration.Tool.CLI/MIGRATION_PROTOCOL_REFERENCE.md [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-|-------------------------------------------------------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| ----------------------------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --- |
```
Comment (Honzaless) [inline Migration.Tool.CLI/MIGRATION_PROTOCOL_REFERENCE.md:12]: The table syntax is broken by adding a column without a heading.
```suggestion
| ----------------------------------------------------------- | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

File: Migration.Tool.Extensions/README.md [hunk 3]
```diff
@@ -1,31 +1,40 @@
...
 
-examples are `Migration.Tool.Extensions/ClassMappings/ClassMappingSample.cs`
+Example code is found in `Migration.Tool.Extensions/ClassMappings/ClassMappingSample.cs`.
 
 ### Class remodeling sample
 
-demonstrated in method `AddSimpleRemodelingSample`, goal is to take single data class and change it to more suitable shape.
+Example code is found in the method `AddSimpleRemodelingSample`.
+
+The goal of this method is to take a **single data class** and change it to more suitable shape.
```
Comment (Honzaless) [inline Migration.Tool.Extensions/README.md:26]: Missing an "a" article.
```suggestion
The goal of this method is to take a **single data class** and change it to a more suitable shape.

File: Migration.Tool.Extensions/README.md [hunk 4]
```diff
@@ -37,68 +46,80 @@ var m = new MultiClassMapping(targetClassName, target =>
...
+Then define a new primary key:
+
 ```csharp
 m.BuildField("EventID").AsPrimaryKey();
 ```
 
-and finally lets define relations to fields:
+Finally, let's define relations to fields:
+
+1. build field title
```
Comment (Honzaless) [inline Migration.Tool.Extensions/README.md:57]: Missing an "a" article.
```suggestion
build a field title

File: Migration.Tool.Extensions/README.md [hunk 5]
```diff
@@ -37,68 +46,80 @@ var m = new MultiClassMapping(targetClassName, target =>
...
+   title.SetFrom("_ET.Event1", "EventTitle", true);
+   // map "EventTitle" field form source data class "_ET.Event2"
+   title.SetFrom("_ET.Event2", "EventTitle");
+
+   // patch field definition, in this case lets change field caption
+   title.WithFieldPatch(f => f.Caption = "Event title");
+   ```
+
+1. in similar fashion map other fields
+1. if needed custom value conversion can be used
```
Comment (Honzaless) [inline Migration.Tool.Extensions/README.md:73]: Consider adding a comma for more clarity and better flow of the text.

File: Migration.Tool.Extensions/README.md [hunk 6]
```diff
@@ -37,68 +46,80 @@ var m = new MultiClassMapping(targetClassName, target =>
...
-            default:
-                // no context is available (possibly when tool is extended with other conversion possibilities)
-                break;
-        }
-
-        return v?.ToString() is { } av && !string.IsNullOrWhiteSpace(av) ? DateTime.Parse(av) : null;
-    });
-startDate.WithFieldPatch(f => f.Caption = "Event start date");
-```
+The goal of this method is to take a **single data class** and assign reusable schema.
```
Comment (Honzaless) [inline Migration.Tool.Extensions/README.md:108]: Missing an "a" article.

File: Migration.Tool.Extensions/README.md [hunk 7]
```diff
@@ -37,68 +46,80 @@ var m = new MultiClassMapping(targetClassName, target =>
...
-```csharp
-serviceCollection.AddSingleton<IClassMapping>(m); 
-```
+### Convert page type to reusable content item (content hub)
 
-### Inject and use reusable schema
+Example code is found in the method `AddReusableRemodelingSample`.
 
-demonstrated in method `AddReusableSchemaIntegrationSample`, goal is to take single data class and assign reusable schema.
+Please note, that all information unique to page will be lost.
```
Comment (Honzaless) [inline Migration.Tool.Extensions/README.md:114]: Missing an "a" article.

File: README.md [hunk 8]
```diff
@@ -10,15 +10,14 @@ This repository is part of the [Xperience by Kentico Migration Toolkit](https://
...
-- [Migration Protocol](./Migration.Toolkit.CLI/MIGRATION_PROTOCOL_REFERENCE.md) -- provides information about the results of the migration and required manual steps, etc.
+Our documentation includes guides covering [the migration process from Kentico Xperience 13 to Xperience by Kentico](https://docs.kentico.com/x/migrate_from_kx13_guides).
 
 ## Library Version Matrix
 
+View all [project releases](https://github.com/Kentico/xperience-by-kentico-kentico-migration-tool/releases/).
+
 | Xperience Version | Library Version |
 | ----------------- | --------------- |
 | 29.6.0            | 1.4.0           |
```
Comment (Honzaless) [inline README.md:23]: Update the version matrix since library version 1.5.0 already exists.

**Pull request merged**